index.js

```diff
@@ -70,21 +71,27 @@ function lint(textEditor) {
 			};
 		}
 
+		// taken from linter-eslint
+		let range;
+		const msgLine = x.line - 1;
+		if (typeof x.endColumn === 'number' && typeof x.endLine === 'number') {
+			const msgCol = Math.max(0, x.column - 1);
+			range = [[msgLine, msgCol], [x.endLine - 1, x.endColumn - 1]];
+		} else if (typeof x.line === 'number' && typeof x.column === 'number') {
+			// We want msgCol to remain undefined if it was initially so
+			// `rangeFromLineNumber` will give us a range over the entire line
+			const msgCol = typeof x.column === 'undefined' ? x.column : x.column - 1;
+			range = rangeFromLineNumber(textEditor, msgLine, msgCol);
```

Comment:
I may be wrong, but I thought Atom has a general try/catch for each package that will result in an error notification and the ability for the user to submit the error to the package. If this is incorrect, please let me know and I will add some sugar over this; otherwise, I think loudly failing on range errors is better than silently failing as `linter-estlint` does. As you mentioned, those bugs should be brought to someones attention, after all.